completers/tox_completer/cmd/exec.go

```diff
@@ -14,30 +16,40 @@ var execCmd = &cobra.Command{
-	execCmd.Flags().Bool("develop", false, "install package in development mode (default: False)")
-	execCmd.Flags().String("exit-and-dump-after", "", "")
+	add_common_flags(execCmd)
+
+
+	execCmd.Flags().String("result-json", "", "write a JSON file with detailed information about all commands and results involved (default: None)")
 	execCmd.Flags().String("hashseed", "", "set PYTHONHASHSEED to SEED before running commands. Defaults to a random integer in the range [1, 4294967295] ([1, 1024] on Windows). Passing 'notset' suppresses this behavior. (default: 2416698050)")
-	execCmd.Flags().BoolP("help", "h", false, "show this help message and exit")
-	execCmd.Flags().String("installpkg", "", "use specified package for installation into venv, instead of packaging the project (default: None)")
+	execCmd.Flags().StringArray("discover", []string{}, "for Python discovery first try these Python executables (default: [])")
 	execCmd.Flags().Bool("list-dependencies", false, "list the dependencies installed during environment setup (default: False)")
 	execCmd.Flags().Bool("no-list-dependencies", false, "never list the dependencies installed during environment setup (default: True)")
-	execCmd.Flags().String("no-provision", "", "do not perform provision, but fail and if a path was provided write provision metadata as JSON to it (default: False)")
-	execCmd.Flags().Bool("no-recreate-pkg", false, "if recreate is set do not recreate packaging tox environment(s) (default: False)")
-	execCmd.Flags().Bool("no-recreate-provision", false, "if recreate is set do not recreate provision tox environment (default: False)")
+	execCmd.Flags().StringS("e", "e", "", "environment to run (default: py)")
+	execCmd.Flags().String("skip-env", "", "exclude all environments selected that match this regular expression (default: '')")
+	execCmd.Flags().StringP("skip-missing-interpreters", "s", "config", "don't fail tests for missing interpreters: {config,true,false} choice (default: config)")
 	execCmd.Flags().BoolP("notest", "n", false, "do not run the test commands (default: False)")
-	execCmd.Flags().StringP("pkg-only,", "b", "", "")
-	execCmd.Flags().BoolP("quiet", "q", false, "decrease verbosity (default: 0)")
-	execCmd.Flags().BoolP("recreate", "r", false, "recreate the tox environments (default: False)")
-	execCmd.Flags().String("result-json", "", "write a JSON file with detailed information about all commands and results involved (default: None)")
-	execCmd.Flags().String("root", "", "project root directory (if not specified will be the folder of the config file) (default: None)")
-	execCmd.Flags().String("runner", "", "the tox run engine to use when not explicitly stated in tox env configuration (default: virtualenv)")
-	execCmd.Flags().String("skip-env", "", "exclude all environments selected that match this regular expression (default: )")
+
+	// these two are the same flag; there are two long flags for this
```

Comment:
That's fine. You can also [mark them mutually exclusive](https://pkg.go.dev/github.com/spf13/cobra@v1.8.1#Command.MarkFlagsMutuallyExclusive) if needed.
So when one is already set the other will not be shown during completion.

Reply:
Thanks! :)